src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap

```diff
@@ -60,9 +60,13 @@ identifiers:
 [(8, 26), (8, 27)] b b
 [(10, 6), (18, 1)] Obj Obj
 [(10, 6), (10, 9)] Obj Obj
+[(11, 2), (11, 5)] 1 foo
 [(12, 2), (12, 9)] doThing doThing
 [(13, 4), (13, 11)] console console
 [(13, 12), (13, 15)] log log
+[(15, 2), (15, 14)] function () {
+  return 42;
+} doOtherThing
```

Comment:
hmm, this seems like a bit much... do we want this? maybe it is fine, just checking...